glossary/public-member-definition.md

```diff
@@ -0,0 +1,9 @@
+---
+type: Resource
+---
+
+# Public Member
+
+A public member is a publicly owned organization that is officially part of, and able to govern, the Foundation for Public Code vereniging. They abide by our articles of association, pay membership dues, and help shape our processes by partaking in the governance and finance of our organization.
+
+*Formal criteria*: signed a membership contract, abides by our articles of association, pays yearly membership dues, and participates in the general assembly.
```

Comment:
I would mention where this comes from. Also I don't think that emphasis is necessary on the 'formal criteria'.

```suggestion
Criteria from [articles of association](../organization/articles-of-association.md): signed a membership contract, abides by our articles of association, pays yearly membership dues, and participates in the general assembly.
```